x-pack/legacy/plugins/infra/public/containers/logs/log_analysis/log_analysis_cleanup.tsx

```diff
@@ -7,7 +7,29 @@
 import createContainer from 'constate-latest';
 import { useMemo } from 'react';
 import { useTrackedPromise } from '../../../utils/use_tracked_promise';
-import { callDeleteJobs, callStopDatafeed } from './api/ml_cleanup';
+import { callDeleteJobs, callStopDatafeed, callGetJobDeletionTasks } from './api/ml_cleanup';
+import { getAllModuleJobIds } from '../../../../common/log_analysis';
+
+const timeout = (ms: number) => new Promise(res => setTimeout(res, ms));
```

Comment:
When working on code I find that the ["stepdown rule"](https://books.google.de/books?id=_i6bDeoCQzsC&lpg=PA37&dq=%22The%20Stepdown%20Rule%22&pg=PA37#v=onepage&q=%22The%20Stepdown%20Rule%22&f=false) really improves the readability. How about we move these lower-level functions beneath the hook function?

Reply:
Yep, fair point. I will move them. I'd considered putting the `timeout` one in utils, but as it has no further uses yet left it where it was.